pkg/datapath/linux/route/route_linux_test.go

```diff
@@ -119,3 +119,22 @@ func (p *RouteSuitePrivileged) TestReplaceRoute(c *C) {
 	// lookup test broken for IPv6 as long as use lo as device
 	testReplaceRoute(c, "f00d::a02:200:0:0/96", "f00d::a02:100:0:815b", false)
 }
+
+func testReplaceRule(c *C, mark, table int) {
+	// delete rule in case it exists from a previous failed run
+	DeleteRule(mark, table)
+	err := ReplaceRule(mark, table)
```

Comment:
I was thinking "weird, you delete the rule then try to replace... shouldn't it fail?", because we typically name "insert or replace" functions as `UpsertXXX` in other parts of the codebase. Would be nice as a separate followup to make this conform, but not necessary for this PR and not a big deal.

Reply:
Seems like a good idea there are a few other cleanups like this so I'll add it to the list.